model/post_list.go

```diff
@@ -95,6 +95,16 @@ func (o *PostList) AddPost(post *Post) {
 	o.Posts[post.Id] = post
 }
 
+func (o *PostList) ExtendAll(other *PostList) {
+	for postId := range other.Posts {
+		o.AddPost(other.Posts[postId])
+	}
+
+	for _, postId := range other.Order {
+		o.Order = append(o.Order, postId)
```

Comment:
Since we're no longer checking if a post is already in the post list, this could add a single post to `o.Order` multiple times. Do you think that could be a problem?

Reply:
For the use case of before and after posts, it will not be a problem since I'd expect those to be unique in both.  But now, I added an extra step to ensure that Order has unique post IDs only.